src/PowerPlaywright.Strategies/Redirectors/RedirectionInfoProvider.cs

```diff
@@ -63,15 +61,10 @@ private async Task<OrgSettings> GetOrgSettingsAsync(IPage page)
         private async Task<AppSettings> GetAppSettingsAsync(IPage page)
         {
             var appSettings = await page.EvaluateAsync(
-                @"(keys) => {
-                    const globalContext = Xrm.Utility.getGlobalContext();
```

Comment:
Hey, mate. I'm not sure if we're going to be able to use `getCurrentAppSettings`. I've checked the docs and this looks like an undocumented part of the API -

- https://learn.microsoft.com/en-us/power-apps/developer/model-driven-apps/clientapi/reference/xrm-utility/getglobalcontext#methods-of-global-context-getglobalcontext
- https://learn.microsoft.com/en-us/power-apps/developer/model-driven-apps/clientapi/get-update-setting-value-using-code

Reply:
Yeah for now until its official we can stick with what we had. I will contact MSFT to see if it will stick around and make the docs